helpers/deploymentConfig.ts

```diff
@@ -773,6 +795,44 @@ export const assets: Assets = {
       oracle: "binance",
     },
   ],
+  arbitrumsepolia: [
+    {
+      token: "WBTC",
+      address: "0xFb8d93FD3Cf18386a5564bb5619cD1FdB130dF7D",
+      price: "68000000000000000000000",
+      oracle: "chainlink",
+    },
+    {
+      token: "USDC",
+      address: "0x86f096B1D970990091319835faF3Ee011708eAe8",
+      price: "1000000000000000000",
+      oracle: "chainlink",
+    },
+    {
+      token: "USDT",
+      address: "0xf3118a17863996B9F2A073c9A66Faaa664355cf8",
+      price: "1000000000000000000",
+      oracle: "chainlink",
+    },
+    {
+      token: "ARB",
+      address: "0x4371bb358aB5cC192E481543417D2F67b8781731",
+      price: "1500000000000000000",
+      oracle: "chainlink",
+    },
+    {
+      token: "WETH",
+      address: "0x980B62Da83eFf3D4576C647993b0c1D7faf17c73",
+      price: "3500000000000000000000",
+      oracle: "chainlink",
+    },
+    // {
+    //   token: "XVS",
+    //   address: "0x47fA6E9F717c9eB081c4734FfB5a1EcD70508891", // TODO: add oracle for XVS once configured
+    //   price: ""
+    //   oracle: ""
+    // }
```

Comment:
I would use the Chainlink oracle for XVS on Arbitrum sepolia, setting a fixed price of $10, for example

Reply:
Done https://github.com/VenusProtocol/oracle/pull/174/commits/5f4f8be3d496f4e7df1ad32a54a4aa6acf5ee422